docs/en/common/module/coffee-module-etcd.adoc

```diff
@@ -49,6 +49,25 @@ even if the ETCD does not contain it,
 so we can reduce a lot of repetitive queries.
 Cache is a thread safe singleton,
 it is possible to clear it by calling `EtcdConfigSourceCache.instance().clear()`.
+* It is possible to disable a ConfigSource with configuration, e.g.:
++
+----
+hu.icellmobilsoft.coffee.module.etcd.producer.CachedEtcdConfigSource.enabled: false
+hu.icellmobilsoft.coffee.module.etcd.producer.DefaultEtcdConfigSource.enabled: false
+hu.icellmobilsoft.coffee.module.etcd.producer.FilteredEtcdConfigSource.enabled: false
+hu.icellmobilsoft.coffee.module.etcd.producer.RuntimeEtcdConfigSource.enabled: false
+----
+You can use the shorter keys too, e.g.:
++
+----
+CachedEtcdConfigSource.enabled: false
+DefaultEtcdConfigSource.enabled: false
+FilteredEtcdConfigSource.enabled: false
+RuntimeEtcdConfigSource.enabled: false
+----
+There are two ways to set this configuration based on the return value of the following methods:
```

Comment:
- így kicsit furcsa nekem a megfogalmazás, vagy a strukturálás, hogy fel vannak sorolva a példák, hogy hogyan lehet kétféleképpen beállítani, aztán újra meg van említve, hogy kétféleképpen be lehet állítani a köv. metódusok által nyújtott kulcsok segítségével.